AI/VCAI/VCAI.cpp

```diff
@@ -2459,6 +2459,25 @@ Goals::TSubgoal VCAI::questToGoal(const QuestInfo & q)
 		}
 		case CQuest::MISSION_HERO:
 		{
+			if(q.obj->tempOwner == playerID)

```

Comment:
MISSION_HERO is not a kill hero quest, see MISSION_KILL_HERO below

Reply:
Ok. Looks like this requires some additional investigation and proper testing.

Reply:
Now it is ok.